<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParportBox</class>
 <widget class="QWidget" name="ParportBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="playout">
        <item>
         <widget class="QToolButton" name="powerButton">
          <property name="text">
           <string>Power</string>
          </property>
          <property name="icon">
           <iconset resource="../../../lib/mainwindow/mainwindow/mw_images.qrc">
            <normaloff>:/mainwindow/system-shutdown.png</normaloff>:/mainwindow/system-shutdown.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="btnData0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btnData2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btnData3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnData1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="btnData4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="btnData5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="btnData6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QPushButton" name="btnData7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelData">
          <property name="text">
           <string>data</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QPushButton" name="btnInvertData">
          <property name="text">
           <string>invert data</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="btnStatusBusy">
          <property name="text">
           <string>Busy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelStatus">
          <property name="text">
           <string>status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnStatusError">
          <property name="text">
           <string>Error</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnStatusSelect">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btnStatusPaperOut">
          <property name="text">
           <string>Paper Out</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="btnStatusACK">
          <property name="text">
           <string>ACK</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnInit">
          <property name="text">
           <string>Init</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btnSelectIn">
          <property name="text">
           <string>Select_In</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="btnStrobe">
          <property name="text">
           <string>Strobe</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="btnAutoFt">
          <property name="text">
           <string>AutoFt</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QPushButton" name="btnInvertControl">
          <property name="text">
           <string>invert control</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelControl">
          <property name="text">
           <string>  control  </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnData0</tabstop>
  <tabstop>btnData1</tabstop>
  <tabstop>btnData2</tabstop>
  <tabstop>btnData3</tabstop>
  <tabstop>btnData4</tabstop>
  <tabstop>btnData5</tabstop>
  <tabstop>btnData6</tabstop>
  <tabstop>btnData7</tabstop>
 </tabstops>
 <resources>
  <include location="../../../lib/mainwindow/mainwindow/mw_images.qrc"/>
 </resources>
 <connections/>
</ui>
